admin/conference-template/process/commentProcess.template

```diff
@@ -21,15 +21,15 @@ function(){
       if(!selfComment){
         if(replytoNote.id == origNote.id){
           author_mail = {
-            "groups": origNote.content.authorids,
-            "subject": "Your submission to "+CONFERENCEPHRASE+" has received a comment",
-            "message": "Your submission to "+CONFERENCEPHRASE+" has received a comment.\n\nComment title: " + note.content.title + "\n\nComment: " + note.content.comment + "\n\nTo view the comment, click here: " + baseUrl+"/forum?id=" + note.forum +'&noteId='+note.id
+            'groups': origNote.content.authorids,
+            'subject': 'Your submission to ' + SHORT_PHRASE +  ' has received a comment',
+            'message': 'Your submission to ' + SHORT_PHRASE + ' has received a comment.\n\nComment title: ' + note.content.title + '\n\nComment: ' + note.content.comment + '\n\nTo view the comment, click here: ' + baseUrl+'/forum?id=' + note.forum +'&noteId='+note.id
           };
         } else {
           author_mail = {
-            "groups": replytoNote.signatures == '(anonymous)' ? [] : replytoNote.signatures,
-            "subject": "Your comment has received a response",
-            "message": "Your comment titled \""+replytoNote.content.title+"\" has received a response.\n\nComment title: " + note.content.title + "\n\nComment: " + note.content.comment + "\n\nTo view the comment, click here: " + baseUrl+"/forum?id=" + note.forum +'&noteId='+note.id
+            'groups': replytoNote.signatures == '(anonymous)' ? [] : replytoNote.signatures,
+            'subject': 'Your comment has received a response',
+            'message': 'Your comment titled \"'+replytoNote.content.title+'\" has received a response.\n\nComment title: ' + note.content.title + '\n\nComment: ' + note.content.comment + '\n\nTo view the comment, click here: ' + baseUrl+'/forum?id=' + note.forum +'&noteId='+note.id
```

Comment:
You don't need to escape the double quote if you used single quotes at the beginning of the string.